src/OpenFga.Sdk/OpenFga.Sdk.csproj

```diff
@@ -33,4 +36,11 @@
     <None Include="../../README.md" Pack="true" PackagePath=""/>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
+    <PackageReference Include="System.Net.Http.Json" Version="7.0.1" />
```

Comment:
These other dependencies should be kept in line with .NET LTS versions as 7.x.x is end of life.